stable/mission-control/UPGRADE_NOTES.md

```diff
@@ -0,0 +1,44 @@
+# JFrog Mission-Control Chart Upgrade Notes
+This file describes special upgrade notes needed at specific versions
+
+## Upgrade from 1.x,2x to 3.X (Chart Versions)
+
+* To upgrade chart version to 3x, you must be on chart version 1.0.5 or above as described in https://github.com/jfrog/charts/blob/master/stable/mission-control/CHANGELOG.md.
+* Data other than your licenses, such as your service information and insight, will not be available after the upgrade.
+* JFrog Mission Control v4.x is only compatible with JFrog Artifactory v7.x. To upgrade, you must first install JFrog Artifactory 7.x.
+* To know more about upgrading mission control, please refer -> https://www.jfrog.com/confluence/display/JFROG/Upgrading+Mission+Control#UpgradingMissionControl-UpgradingfromVersion3.5.1to4.x
+
+**DOWNTIME IS REQUIRED FOR AN UPGRADE!**
+
+* Data export is done with a migration script jfmcDataExport.sh (available in the other installers. Please get the file from https://jfrog.com/download-jfrog-platform). You can obtain the file from the docker compose installer.
```

Comment:
@rahulsadanandan Can we add this `jfmcDataExport.sh`  file  under chart ?

Reply:
Its now added under files directory in mission-control chart